apps/api/v2/src/ee/event-types/event-types_2024_04_15/outputs/get-event-type-public.output.ts

```diff
@@ -399,8 +399,8 @@ class PublicEventTypeOutput {
   workflows!: any[];
 
   @IsArray()
-  @ApiProperty()
-  hosts!: any[];
+  @ApiPropertyOptional()
+  hosts?: any[];
```

Comment:
I know i am late to the party and it's merged, but it's best to keep hosts as an array but just return it empty - it's called a `null` pattern, because now wherever code depends on array and accesses something at index 0 could break.